_posts/2018-12-18-downloader.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: Initial release of OMERO.downloader
+categories: tools
+---
+
+# Introducing OMERO.downloader
+
+For OMERO to properly fulfill the role of being a useful repository for
+microscopy images its users must have easy access to that data. As data
+sets grow in size it becomes a correspondingly greater challenge to
+provide access to that data. This motivates the creation of server-side
+solutions such as the IDR's
+[Virtual Analysis Environment](https://idr-analysis.openmicroscopy.org/).
+
+For the past couple of years the OME team has been investigating ways to
+improve users' ability to obtain data from OMERO for client-side storage
+and processing. We now release OMERO.downloader v0.1.0 which is still an
+early version missing many features but should already add significant
+value for some use cases.
+
+OMERO.downloader is a Java application that acts as a command-line OMERO
+client. It writes selected data from an OMERO server into a local
+directory and creates soft links to represent some of the metadata
+linking the server objects.
+
+OMERO.downloader is designed to handle situations in which not all the
```

Comment:
It will be good to indicate that downloader will not try to re-download if the file is there.
This question is often asked